Docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3.9'
+services:
+  postgres:
+    image: 'postgres:14.2-alpine'
+    ports:
+      - '5432:5432'
+    environment:
+      POSTGRES_PASSWORD: 'mysecretpassword'
```

Comment:
Better use env https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option